tributors/main/parsers/mailmap.py

```diff
@@ -38,9 +38,21 @@ def load_data(self):
                 sys.exit("%s does not exist" % self.filename)
 
             for line in read_file(self.filename):
-                name, email = line.split("<")
-                email = email.strip().rstrip(">")
-                self.data[email] = {"name": name.strip()}
+
+                # keep track of the previous name, in case multiple per line
+                name = None
+
+                # mailmap line can have more than one entry, split by right >
+                for entry in line.strip().split(">"):
+                    if not entry:
+                        continue
+                    new_name, email = map(str.strip, entry.split("<"))
+                    # only the first name matters
+                    if not name and new_name:
+                        name = new_name
+                    if not name:
```

Comment:
Do we really want an error raised here?

Reply:
Explicit better than implicit... Baku formatted file better leads to error than wrong result IMHO